.travis.yml

```
@@ -11,8 +11,8 @@ before_install:
 - MAX_LINE_LENGTH=100 bash ci/check_line_lengths.sh src/**/*.md
 install:
 - source ~/.cargo/env || true
-- cargo install mdbook --version 0.4.1
-- cargo install mdbook-linkcheck --version 0.7.0
+- cargo install mdbook --version '^0.4.3'
+- cargo install mdbook-linkcheck --version '^0.7.0'
```

I can skip having a caret requirement for `mdbook-linkcheck` if you want to manually update it. I don't think it's updated that often anyway.

```suggestion
- cargo install mdbook-linkcheck --version 0.7.0
```